tests/failpoints/cases/test_merge.rs

```diff
@@ -279,6 +279,7 @@ fn test_node_merge_multiple_snapshots_together() {
 // }
 
 fn test_node_merge_multiple_snapshots(together: bool) {
+    let _guard = ::setup();
```

Comment:
Please remove this line.

Reply:
No, it's intended. Every failpoint test cases should include this line.